ginga/canvas/types/basic.py

```diff
@@ -158,7 +159,7 @@ def contains_arr(self, x_arr, y_arr):
         # See: http://alienryderflex.com/polygon/
         xa, ya = x_arr, y_arr
 
-        result = numpy.empty(ya.shape, dtype=numpy.bool)
+        result = numpy.empty((ya.size, xa.size), dtype=numpy.bool)
```

Comment:
@pllim, this change breaks the basic polygon containment test.  I'm pretty certain the result is supposed to be a 1D array of bools that is as long as the number of coordinates being tested--that length is `ya.shape` or `xa.shape` (they should be equivalent).  This would be consistent with the other canvas items.

Can you describe the case where it needs to be otherwise?